modules/nested-tabs/assets/scss/frontend.scss

```
@@ -326,13 +326,24 @@
 	background-image: initial;
 }
 
+.elementor .elementor-element.elementor-widget-n-tabs:not(:has(> .elementor-widget-container)) > .e-n-tabs[data-touch-mode="false"] > .e-n-tabs-heading .e-n-tab-title[aria-selected="false"]:hover {
```

I am thinking can we remove the :`not()` part, because we are already using these selector `>`

```suggestion
.elementor .elementor-element.elementor-widget-n-tabs > .e-n-tabs[data-touch-mode="false"] > .e-n-tabs-heading .e-n-tab-title[aria-selected="false"]:hover {
```

---

@hein-obox Not sure what you mean here. We need the not to make sure this CSS applies when the elementor-widget-container div is not present.

---

@louiswol94 You are right! You can ignore this comment.